docs-source/content/userguide/managing-domains/model-in-image/runtime-updates.md

```diff
@@ -0,0 +1,325 @@
+
+ - For simple ways to change `restartVersion`, see [Changing a domain resource 'restartVersion'](#changing-a-domain-resource-restartversion).
+
+ - For a sample of adding a data source to a running domain, see [Example of adding a data source](#example-of-adding-a-data-source).
+
+ - For a discussion of model file syntax, see the [WebLogic Deploy Tool](https://github.com/oracle/weblogic-deploy-tooling) documentation and [Model files]({{< relref "/userguide/managing-domains/model-in-image/model-files.md" >}}).
+
+ - If the introspector job reports a failure, see [Debugging]({{< relref "/userguide/managing-domains/model-in-image/debugging.md" >}}) for debugging advice.
+
+#### Frequently asked questions
+
+_Why is it necessary to specify updates using model files?_
+
+Similar to Domain in Image, if you make a direct runtime WebLogic configuration update of a Model in Image domain using the WebLogic console or WLST scripts, then the update is ephemeral. This is because the domain home is stored in an image directory which will not survive the restart of the owning pod.
+
+_How do Model in Image updates work during runtime?_
+
+After you make a change to your domain resource `restartVersion` or `image` attribute, the operator will rerun the domain's introspector job. This job will reload all of your secrets and environment variables, merge all of your model files, and generate a new domain home. If the job succeeds, then the operator will make the updated domain home available to pods using a config map named `DOMAIN_UID-weblogic-domain-introspect-cm`. Finally, the operator will subsequently roll (restart) each running WebLogic Server pod in the domain so that it can load the new configuration. A domain roll begins by restarting the domain's Administration Server and then proceeds to restart each Manager Server in the domain.
+
+_Can we use custom configuration overrides to do the updates instead?_
+
+No. Custom configuration overrides, which are WebLogic configuration overrides specified using a domain resource `configuration.overridesConfigMap`, as described in [Configuration overrides]({{< relref "/userguide/managing-domains/configoverrides/_index.md" >}}), aren't supported in combination with Model in Image. Model in Image will generate an error if custom overrides are specified. This should not be a concern because model file, secret, and/or model image updates are simpler and more flexible than custom configuration override updates. Unlike configuration overrides, the syntax for a model file update exactly matches the syntax for specifying your model file in the first place.
+
+
+#### Supported and unsupported updates
+
+ - You can add new MBeans or resources simply by specifying their corresponding model file YAML snippet along with their parent bean hierarchy. See [Example of adding a data source](#example-of-adding-a-datasource).
```

Comment:
Do we want to spell out deploying new application/archives use case (technically a mbean)?

Reply:
@jshum2479 No, we don't.  We just implicitly that you can deploy a new image, which implicitly includes the applications with the image.  

What specifically do you think we should state here?

Reply:
I am thinking about stating clearly the application archives (hence app/shared libs etc) needs to be on the image, other mbeans resources can be on the configmap only, but if you think it's clear enough then we can close this.